### PR TITLE
Amendment

### PR DESCRIPTION
**Pull request by Kyle-Law**
Youtube Clone completed. It's responsive.
Live link deployed: https://rawcdn.githack.com/Kyle-Law/Youtube-Clone-Daniel-Kyle/b6fab969116a2aba4bb920a635bff3be94b29c02/index.html

### CODE REVIEW

**Comment by kobiyoyo** [comment]
### Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team 
#### Great Job so far, but there few changes required. 

### Page structure
- [x] There must be a share button
- [x] It shows the avatar of the user who uploaded the video
- [x] It has a subscribe button



### Make changes and submit another code review request.
*Happy coding*

**Review by nikasulo — CHANGES_REQUESTED** [review]
# This is amazing Work! 
One tiny issue, your stickler errors persist. Please fix all errors and request another review. I wish you the best 😊.

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: styles.css [hunk 1]
```diff
@@ -75,54 +90,91 @@ main{
...
-#video-details > ul >li{
-    list-style: none;
-    display: grid;
-    grid-template-columns: repeat(2,1fr);
+
+.video-details > ul >li{
     align-items: center;
     column-gap: 5px;
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
```
[the remark this answers is not in the document]
Comment (Kyle-Law) [reply styles.css]: Tried but unable to resolve RBRACE error.

File: styles.css [hunk 2]
```diff
@@ -75,54 +90,91 @@ main{
...
+/* Biography Section */
+.biography{
+    display: grid;
+    grid-template-columns: auto 120px;
+    margin-top: 20px;
+}
+
+.author{
+    display: grid;
+    grid: repeat(2,1fr) / 60px 1fr;
```
[the remark this answers is not in the document]
Comment (Kyle-Law) [reply styles.css]: Tried but unable to resolve RBRACE error.

**Comment by Kyle-Law** [comment]
Tried but unable to resolve RBRACE error. They are the last 2 lint errors that we can't resolve.

**Comment by simondebbarma** [comment]
Hi guys,

Your .stickler.yml file is configured incorrectly. You have it configured for bower instead of stylelint.

```
linters:
  stylelint:
    config: 'stylelint.config.js'
```

**Review by nikasulo — APPROVED** [review]
# Fantastic work @Kyle-Law and @simonpeterdebbarma!
I saw your message on Twitter about needing some assistance, apologies for not responding on time, trying to meet up with the deadline for my final capstone project so I was off social media the whole day. However, I will send you my contact so you can reach me at anytime subsequently if you need assistance.

All that being said, this is awesome guys! You deserve nothing short of an approval! Please merge and move to exfil! Bravo-one out :rocket:

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by simondebbarma** [comment]
@Oluwadamilareolusakin I wasn't part of this project. I'm a mentor of the other guy who worked on this, @Daniduran-dev . They had some stickler issue so I just dropped by to help. Would you kindly untag me from your comment and credit Daniel who worked on this instead! Thanks.

**Comment by Kyle-Law** [comment]
Thanks @simonpeterdebbarma for the helping hand! It's a lifesaver.
Thanks @Oluwadamilareolusakin for reviewing our project 👍  Good luck to your capstone!

**Pull request merged**